webknossos/webknossos/dataset/layer.py

```diff
@@ -762,13 +774,14 @@ def downsample_mag(
 
         # Get target view
         target_view = target_mag_view.get_view(
-            offset=aligned_target_bb.topleft,
-            size=aligned_target_bb.size,
+            absolute_offset=aligned_target_bb.topleft
```

Comment:
Is this the right way of doing it? @jstriebel

Reply:
For now yes, we'll add the bounding-box arguments for `get_view` as well, later, but it was quite complex with the backwards-compatible deprecations.